unitex/src/fr/umlv/unitex/leximir/model/DictionaryPath.java

```diff
@@ -29,6 +29,8 @@
  */
 public class DictionaryPath {
     public static ArrayList<String> dictionary= new ArrayList<>();
+    public static final String statisticsDelasPath = Config.getUserDir().toString()+File.separator+Config.getCurrentLanguage()+File.separator+"Dela"+File.separator+"Delas"+File.separator+"Statistics"+File.separator;
+    public static final String statisticsDelacPath = Config.getUserDir().toString()+File.separator+Config.getCurrentLanguage()+File.separator+"Dela"+File.separator+"Delac"+File.separator+"Statistics"+File.separator;   
     public static final String allDelafAbsPath = Config.getUserDir().toString()+File.separator+Config.getCurrentLanguage()+File.separator+"Dela"+File.separator;
     public static final String delafTmpPathDelac = Config.getUserDir().toString()+File.separator+Config.getCurrentLanguage()+File.separator+"text.txt";
```

Comment:
What is the purpose of this file? There is a reason to hard-coded its name?

Reply:
This is a folder that contains the results of the statistics on a DELAS or DELAC. It was suggested by Cvetana Krstev to add it so the statistic.csv file has a predefined path. This folder will also contains in the futur the result of the duplicates finding.

Reply:
I'm asking about `text.txt`

Reply:
I don't know what is its use yet, it was in Anas code and it seems like it is used for the DELAC edition and I haven't worked on it yet. I'll look for what it does when I get to the DELAC edition.

Reply:
@aissabnhd, a comment about this?

Reply:
This field is only used in the function "generateDelaf" in unitex/src/fr/umlv/unitex/leximir/util/Utils.java and this function is never called in the program. It seems to be useless because when we comment the field and the function, there is no problem. Maybe we don't need them and we should remove them from the code ?